include/tesseract/publictypes.h

```diff
@@ -278,6 +278,17 @@ enum OcrEngineMode {
   OEM_COUNT                    // Number of OEMs
 };
 
+/** 
+ * Except when OTSU_TRESH is chosen
+ * Leptonica is used for thresholding 
+ */
+enum ThreshMethod {
```

Comment:
enum class?

Reply:
Sure, I can change it to enum class.

about style:

`OTSU_THRESH` ->
`otsu` ? 
`Otsu` ?
(droping the _thresh in both cases).

Reply:
I think it should be Otsu.
You can also grep other `enum class`  vars in tess.

Reply:
https://github.com/tesseract-ocr/tesseract/search?q=%22enum+class%22&type=

'kName' is used in 3 out of 4 enum class cases.
This is google style. Should I also use it or should I drop the 'k' prefix?

Reply:
No.

Reply:
>Should I also use it or should I drop the 'k' prefix?

> No.

No to which part of the question?

 1) Should I also use it
2) or should I drop the 'k' prefix?

Reply:
Without `k`. Just `Otsu` etc.